index.bs

```diff
@@ -6438,6 +6445,90 @@ requests will be affected.
 
 </div>
 
+#### The network.setCacheMode Command ####  {#command-network-setCacheMode}
+
+The <dfn export for=commands>network.removeSetCacheMode</dfn> command overrides
+the network cache behaviour for certain requests.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      network.SetCacheMode = (
+        method: "network.setCacheMode",
+        params: network.SetCacheModeParameters
+      )
+
+      network.SetCacheModeParameters = {
+        mode: "no-store" / null,
+        ? contexts: [+browsingContext.BrowsingContext]
+      }
+    </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl">
+      EmptyResult
+    </pre>
+   </dd>
+</dl>
+
+<div algorithm>
+The <dfn export>WebDriver BiDi update request cache mode</dfn> steps given
+|request| are:
+
+1. Let |context| be null.
+
+1. If |request|'s [=request/window=] is an [=environment settings object=]:
+
+  1. Let |environment settings| be |request|'s [=request/window=]
+
+  1. If there is a [=/browsing context=] whose [=active window=] is |environment
+     settings|' [=environment settings object/global object=], set |context| to
+     the [=top-level browsing context=] for that context.
+
+1. If |context| is not null:
+
+  1. If [=cache mode override map=] [=map/contains=] |context|, set
+     |request|&apos;s [=request/cache mode=] to [=cache mode override
+     map=][|context|] and return.
+
+1. If [=default cache mode override=] is not null, set
+     |request|&apos;s [=request/cache mode=] to [=default cache mode override=].
+
+</div>
+
+<div algorithm="remote end steps for network.setCacheMode">
+The [=remote end steps=] given <var ignore>session</var> and |command parameters| are:
+
+1. Let |mode| be |command parameters|["<code>mode</code>"].
+
+1. If |command parameters| does not [=map/contain=] "<code>contexts</code>", set
+   the [=default cache mode override=] to |mode| and return [=success=] with
+   data null.
+
+1. Let |contexts| be an empty [=/list=].
+
+1. For each |context id| of |command parameters|["<code>contexts</code>"]:
+
+  1. Let |context| be the result of [=trying=] to [=get a browsing context=]
```

Comment:
This should be [similar to what we already have for `network.addIntercept`](https://w3c.github.io/webdriver-bidi/#command-network-addIntercept) so that we use a `Set` and can filter out identical browsing contexts.

Reply:
It doesn't really matter (here, or afaict in `addIntercept`) because it's idempotent, so processing the same context twice doesn't change the outcome. Indeed the `addIntercept` text wrongly suggests it's possible to get an empty list of contexts (probably because of a CDDL change). Anyway, unless we were planning to change things so that passing a context twice is actually an error, I'd prefer to leave things as they are.